web.py

```diff
@@ -158,16 +158,34 @@ def get_pokemarkers():
     session.close()
 
     for pokemon in pokemons:
-        markers.append({
-            'id': 'pokemon-{}'.format(pokemon.id),
-            'type': 'pokemon',
-            'trash': pokemon.pokemon_id in config.TRASH_IDS,
-            'name': POKEMON_NAMES[pokemon.pokemon_id],
-            'pokemon_id': pokemon.pokemon_id,
-            'lat': pokemon.lat,
-            'lon': pokemon.lon,
-            'expires_at': pokemon.expire_timestamp,
-        })
+        if (pokemon.atk_iv is None):
+            markers.append({
+                'id': 'pokemon-{}'.format(pokemon.id),
```

Comment:
Take a look at how https://github.com/evenly-epic-mule/pokeminer/blob/webhooks/notification.py#L662 is done
some think like
```
marker = { [...] }
if config.MAP_SHOW_IV:
    try:
        [...]
    except KeyError:
            pass
markers.append(marker)
```